x-pack/solutions/security/plugins/security_solution/public/detections/components/alerts_table/index.tsx

```diff
@@ -127,7 +128,7 @@ interface AlertTableProps
   extends SetOptional<SecurityAlertsTableProps, 'id' | 'ruleTypeIds' | 'query'> {
   inputFilters?: Filter[];
   tableType?: TableId;
-  sourcererScope?: PageScope;
+  pageScope?: PageScope;
```

Comment:
Thank you making this change. While not necessary as part of this PR, it is definitely welcomed!